lib/utils/format.js

```diff
@@ -0,0 +1,14 @@
+export default (function() {
```

Comment:
You don't need to export anything here, or even use a self-executing function. Simply importing into lib/giantbot.js will run anything in the file.